src/components/ProjectSettingsPage/index.jsx

```diff
@@ -310,10 +310,10 @@ class ProjectSettingsPage extends React.Component {
                           <div className={styles.DeleteSubDescription}>
                             This will permanantly delete the project and all
                             resourses it contains.
-                          </div>
-                          <div>
+                            <div>
```

Comment:
@khalifan-kfan What of "This will permanently delete the project and all its resources."? Fixes a few typos in there too.

Reply:
True

Reply:
![typo fix](https://user-images.githubusercontent.com/69305400/126945371-9c81be64-7fa5-4b2e-8eff-872024e846ae.png)
I think that fixes the issue

Reply:
@khalifan-kfan bro the word is "permanently".

Reply:
Ohh my bad let me get it

Reply:
I have changed it

Reply:
on the same paragraph as required
![same paragraph](https://user-images.githubusercontent.com/69305400/127152145-6fc1e7c7-38f6-497a-af47-a66f18f3a65e.png)